rfcs/models/infrastructure/relays.md

```diff
@@ -73,11 +72,32 @@ Once a driver is designed for a given DLT, it can be reused in any network built
+    
+- **GetState(networks.networks.GetStateMessage): returns common.state.RequestState**
+    This RPC endpoint is used by client application for polling the local relay to get the response to the query by using `request_id` returned in previous API. The local relay fetches the database state based on provided `request_id` and prepares the response. It takes a message of type [GetStateMessage](../../formats/communication/relay.md#getstatemessage) as argument, and returns [RequestState](../../formats/views/request.md#requeststate) message.
+
+### API for other Relays
+
+- **RequestState(common.query.Query): returns common.ack.Ack**
+    The requesting relay sends a data sharing request, using RequestState API, to the remote relay with a query defining the data it wants to receive. The remote relay calls the driver using the gRPC endpoint provided by driver server (RequestDriverState, details below), passing on the query to the driver. It takes a message of type [Query](../../formats/views/request.md#query) as argument, and returns [Ack](../../formats/views/request.md#ack) message.
+
+- **SendState(common.state.ViewPayload): returns common.ack.Ack**
+    The remote relay, after receiving the response from the driver, asynchronously sends it to requesting relay using this RPC endpoint. The requesting relay upon receiving the response, stores it in the database corresponding to the `request_id` key. It takes a message of type [ViewPayload](../../formats/views/request.md#viewpayload) as argument, and returns [Ack](../../formats/views/request.md#ack) message.
+
+### API for Driver
+
+- **SendDriverState(common.state.ViewPayload): returns common.ack.Ack**
+    This RPC endpoint is used by the driver to send back the response of the query, to the remote relay. The remote relay then passes this response to the requesting relay using previous API (SendState). It takes a message of type [ViewPayload](../../formats/views/request.md#viewpayload) as argument, and returns [Ack](../../formats/views/request.md#ack) message.
+
 
 ## Driver Functional API
 
-_TBD: List API function specs exposed to relays. Message formats will be described in 'formats/communication'_
+- **RequestDriverState(common.query.Query): returns (common.ack.Ack)**
+    The remote relay sends a RequestDriverState request to its driver with a query defining the data it wants to receive. This is where driver unpacks the view address, makes a call to the requested contract using given arguments, and collects the response along with the proof, packages them into a DLT-specific view message, and then encapsulating it to a DLT-neutral view message. This final message is then sent back to remote relay using above relay's API (SendDriverState). It takes a message of type takes [Query](../../formats/views/request.md#query) as argument, and returns [Ack](../../formats/views/request.md#ack).
```

Comment:
Don't think you need to use the keyword "RequestDriverState" in the body, as it is already highlighted above. If you do want to use it, wrap it in quotes.